runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -28,14 +28,7 @@
 import org.corfudb.runtime.exceptions.WrongClusterException;
 import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
 import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuInterruptedError;
-import org.corfudb.runtime.view.AddressSpaceView;
-import org.corfudb.runtime.view.Layout;
-import org.corfudb.runtime.view.LayoutManagementView;
-import org.corfudb.runtime.view.LayoutView;
-import org.corfudb.runtime.view.ManagementView;
-import org.corfudb.runtime.view.ObjectsView;
-import org.corfudb.runtime.view.SequencerView;
-import org.corfudb.runtime.view.StreamsView;
+import org.corfudb.runtime.view.*;
```

Comment:
Can you remove the wildcard?

Reply:
That is coming from the IDE not me. If the wildcard should not exist then I have to find a way to trick Intellij. But curious to know why do we want to remove the wildcard, the runtime anyway imports anything in the view right?

Reply:
Done. Got rid of the wildcard anyway.

Reply:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports